web-client/iron-remote-desktop/src/services/clipboard.service.ts

```diff
@@ -0,0 +1,312 @@
+import type { RemoteDesktopService } from './remote-desktop.service';
+import { isComponentDestroyed } from '../lib/stores/componentLifecycleStore';
+import { get } from 'svelte/store';
+import type { ClipboardData } from '../interfaces/ClipboardData';
+import type { RemoteDesktopModule } from '../interfaces/RemoteDesktopModule';
+import { runWhenFocusedQueue } from '../lib/stores/runWhenFocusedStore';
+import { SessionEventType } from '../enums/SessionEventType';
+
+const CLIPBOARD_MONITORING_INTERVAL = 100; // ms
```

Comment:
suggestion: If using a stronger type (such as Duration) is not an option, I would go for a longer naming which includes the unit
```suggestion
const CLIPBOARD_MONITORING_INTERVAL_MS = 100;
```

Reply:
Done in https://github.com/Devolutions/IronRDP/pull/955